packages/@guardian/source-foundations/src/typography/data.ts

```diff
@@ -1,241 +1,149 @@
 import { pxToRem } from '../utils/px-to-rem';
-import type { Category, FontWeight, FontWeightDefinition } from './types';
 
-const fontSizes = [12, 14, 15, 17, 20, 24, 28, 34, 42, 50, 70] as const;
-
-const fonts = {
-	titlepiece: 'GT Guardian Titlepiece, Georgia, serif',
-	headlineSerif: 'GH Guardian Headline, Guardian Egyptian Web, Georgia, serif',
-	bodySerif: 'GuardianTextEgyptian, Guardian Text Egyptian Web, Georgia, serif',
-	bodySans:
-		'GuardianTextSans, Guardian Text Sans Web, Helvetica Neue, Helvetica, Arial, Lucida Grande, sans-serif',
-} as const;
-
-const lineHeights = [1.15, 1.35, 1.5] as const;
-
-const fontWeights = [300, 400, 500, 700] as const;
-
-const underlineThickness = [2, 3, 4, 5, 6] as const;
-
-const titlepieceSizes = {
-	small: fontSizes[8], //42px
-	medium: fontSizes[9], //50px
-	large: fontSizes[10], //70px
-} as const;
-
-const headlineSizes = {
-	xxxsmall: fontSizes[3], //17px
-	xxsmall: fontSizes[4], //20px
-	xsmall: fontSizes[5], //24px
-	small: fontSizes[6], //28px
-	medium: fontSizes[7], //34px
-	large: fontSizes[8], //42px
-	xlarge: fontSizes[9], //50px
-} as const;
-
-const bodySizes = {
-	xsmall: fontSizes[1], //14px
-	small: fontSizes[2], //15px
-	medium: fontSizes[3], //17px
-} as const;
-
-const textSansSizes = {
-	xxsmall: fontSizes[0], //12px
-	xsmall: fontSizes[1], //14px
-	small: fontSizes[2], //15px
-	medium: fontSizes[3], //17px
-	large: fontSizes[4], //20px
-	xlarge: fontSizes[5], //24px
-	xxlarge: fontSizes[6], //28px
-	xxxlarge: fontSizes[7], //34px
-} as const;
-
-const fontSizeMapping = {
-	titlepiece: titlepieceSizes,
-	headline: headlineSizes,
-	body: bodySizes,
-	textSans: textSansSizes,
-} as const;
-
-const remFontSizes = fontSizes.map((fontSize) => pxToRem(fontSize));
-
-const remTitlepieceSizes = {
-	small: remFontSizes[8], //42px
-	medium: remFontSizes[9], //50px
-	large: remFontSizes[10], //70px
-} as const;
-
-const remHeadlineSizes = {
-	xxxsmall: remFontSizes[3], //17px
-	xxsmall: remFontSizes[4], //20px
-	xsmall: remFontSizes[5], //24px
-	small: remFontSizes[6], //28px
-	medium: remFontSizes[7], //34px
-	large: remFontSizes[8], //42px
-	xlarge: remFontSizes[9], //50px
-} as const;
-
-const remBodySizes = {
-	xsmall: remFontSizes[1], //14px
-	small: remFontSizes[2], //15px
-	medium: remFontSizes[3], //17px
-} as const;
-
-const remTextSansSizes = {
-	xxsmall: remFontSizes[0], //12px
-	xsmall: remFontSizes[1], //14px
-	small: remFontSizes[2], //15px
-	medium: remFontSizes[3], //17px
-	large: remFontSizes[4], //20px
-	xlarge: remFontSizes[5], //24px
-	xxlarge: remFontSizes[6], //28px
-	xxxlarge: remFontSizes[7], //34px
+/**
+ * Pixel size values for each font that we use in the design system.
+ *
+ * These fit the following px size scale:
+ * [12, 14, 15, 17, 20, 24, 28, 34, 42, 50, 70]
```

Comment:
Did we wonder about removing these commented values?

Reply:
yep, missed this 👍